dotCMS/src/main/java/com/dotmarketing/portlets/rules/util/RulesImportExportUtil.java

```diff
@@ -83,69 +69,79 @@ public void export(File file) throws IOException {
 		}
 	}
 
-	/**
-	 * 
-	 * @param file
-	 * @throws IOException
-	 */
 	public void importRules(File file) throws IOException {
 
-		RulesAPI rapi = APILocator.getRulesAPI();
+		this.importRules(new FileReader(file));
+	}
+
+	public void importRules(final Reader reader) throws IOException {
+
+		final ObjectMapper mapper = new ObjectMapper();
+		final StringWriter stringWriter = new StringWriter();
+		BufferedReader bufferedReader = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Ensure that resources like this BufferedReader object are closed after use](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4925354)